src/main/java/mate/jdbc/dao/CarDaoImpl.java

```diff
@@ -0,0 +1,226 @@
+package mate.jdbc.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.exception.DataProcessingException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class CarDaoImpl implements CarDao {
+    @Override
+    public Car create(Car car) {
+        String query = "INSERT INTO cars (model, manufacturer_id) VALUES (?, ?)";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement insertCarStatement
+                        = connection.prepareStatement(query,
+                        Statement.RETURN_GENERATED_KEYS)) {
+            insertCarStatement.setString(1, car.getModel());
+            insertCarStatement.setLong(2, car.getManufacturer().getId());
+            insertCarStatement.executeUpdate();
+            ResultSet resultSet = insertCarStatement.getGeneratedKeys();
+            while (resultSet.next()) {
+                car.setId(resultSet.getObject(1, Long.class));
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Couldn't insert car " + car + " to DB", e);
+        }
+        addDriverToCar(car);
+        return car;
+    }
+
+    @Override
+    public Optional<Car> get(Long id) {
+        String query = "SELECT c.id, c.model, c.manufacturer_id, m.name, m.country "
+                + "FROM taxi_services.cars c "
+                + "JOIN taxi_services.manufacturers m "
```

Comment:
```suggestion
                + "FROM cars c "
                + "JOIN manufacturers m "
```

is it works without specifying shema?